tests/unit/entrypoint/database/test_parse.py

```diff
@@ -134,3 +134,33 @@ def test_pubmed_xml_set(tmp_path):
             data = json.load(f)
             loaded_uid = data["uid"]
             assert loaded_uid == uid
+
+
+def test_dry_run(capsys):
+    input_path = "tests/data/cord19_v35/"
+    main(["parse", "cord19-json", input_path, "parsed/", "--dry-run"])
```

Comment:
Just a comment, it's probably not originating from this PR anyway, but I'm wondering if it's useful to go through the main entry-point when testing subcommands. An alternative approach to testing is in `test_convert_pdf` where you only ever refer to the subcommand module.

Reply:
This is indeed `not` originating from this PR.

The current approach follows what @jankrepl started.

Anyway, feel free to open a ticket to get the rationale from @jankrepl and maybe come to a change if needed.